### PR TITLE
Updating README.md file

### LINKED ISSUE #31: Updating ReadMe file
- Update the Team members list of the ReadMe.md file

### PR DESCRIPTION
**Pull request by Leeadar**
- Updating the team members
- Adding a new line between each of the team members

closes #31

### CODE REVIEW

**Review by maayanhd — COMMENTED** [review]
if you already opened a new PR, somehow the new lines were messed up after one of the fixes in the last PR I opened (related to #12), so I suggest also adding a new line before the "NOTE" in the Contribution part, would be helpful

**Comment by Leeadar** [comment]
I made the change, is it good now? @maayanhd

**Review by maayanhd — APPROVED** [review]

**Review by AmirMiller91 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -55,4 +54,4 @@ language
    3. **Commit** changes to your own branch
    4. **push your work back up to your forked repo**
    5. Submit a Pull Request so that we can review your changes
```
Comment (kobihk) [inline README.md:56]: Please add empty line between line #56 to line #57 to make each line separated

**Review by kobihk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Leeadar) [reply README.md:56]: I added the empty line

**Review by kobihk — APPROVED** [review]

**Pull request merged**